blazor/mention/code-snippet/web-api-adaptor.razor

```diff
@@ -0,0 +1,42 @@
+@using Syncfusion.Blazor.DropDowns
+@using Syncfusion.Blazor.Data
+
+<SfMention TItem="EmployeeData" Query="@Query" PopupWidth="200px" suggestionCount=6>
+    <TargetComponent>
```

Comment:
suggestionCount=6>  --> remove it?  Check the all property casing

Reply:
If we removed the Suggestion count, the scrollbar will appears in the demo image so not removed the proeprty. Casing issue resolved.